.github/workflows/publish-docs.yml

```diff
@@ -7,7 +7,7 @@ on:
       - '!master'
 
 env:
-  node-version: 12.x
+  node-version: 14.x
   python-version: 3.x
```

Comment:
@viceice The current actions use Python `3.9.1` according to the latest GitHub Action logs.

```
Run actions/setup-python@v2.2.1
  with:
    python-version: 3.x
    token: ***
  env:
    node-version: 12.x
    python-version: 3.x
Successfully setup CPython (3.9.1)
```